examples/react-native/README.md

```diff
@@ -1,25 +1,21 @@
-This examples shows how to use `react-instantsearch` along with `react-native`.
+# example-react-native
 
-It was created using `create-react-native-app`.
+_This project was generated with [create-instantsearch-app](https://github.com/algolia/create-instantsearch-app) by [Algolia](https://algolia.com)._
 
-## Clone the example
+This examples shows how to use `react-instantsearch` along with `react-native`.
 
-```
-curl https://codeload.github.com/algolia/react-instantsearch/tar.gz/master | tar -xz --strip=2 react-instantsearch-master/examples/react-native
-```
+## Get started
 
-## Start the example
+To run this project locally, install the dependencies and run the local server:
 
 ```sh
-yarn install --no-lockfile
-
-yarn start //to see the app on your phone using the expo client
```

Comment:
this information seems to be gone

Reply:
Which one? We still have the `yarn start` command in the README coming from CISA.

Reply:
The lower ones, like `start ios` and the comment about expo

Reply:
I added them back!